src/styles/blocks/card.scss

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 406px;
+  width: 198px;
```

Comment:
```suggestion
  width: 200px;
```

Reply:
I should then add box-sizing: border-box; to this element?